metadata/org.hibernate.orm/hibernate-core/index.json

```diff
@@ -1,6 +1,14 @@
 [
   {
     "latest": true,
+    "metadata-version": "6.5.0.Final",
+    "default-for": "6\\.5\\..*",
+    "module": "org.hibernate.orm:hibernate-core",
+    "tested-versions": [
+      "6.5.0.Final"
+    ]
+  },
+  {
     "metadata-version": "6.2.0.Final",
```

Comment:
Should we add a `default-for` entry to target 6.2, 6.3 and 6.4 with those `6.2.0.Final` metadata to avoid breaking those versions?

Reply:
yeah, that makes sense. thanks for pointing that out @sdeleuze !

Reply:
Should `"default-for": "6\\.5\\..*",` be removed from the latest metadata version, since the latest metadata version needs to be used for all future hibernate versions until new metadata version get created?

Reply:
Indeed, @christophstrobl could you please remove the `"default-for": "6\\.5\\..*"` entry for the 6.5 metadata where `"latest": true,` should be enough?

Maybe we can also change `"default-for": "6\\.[2-4]\\..*",` to `"default-for": "6\\.[0-4]\\..*",` for the 6.2 metadata in order to avoid breaking Hibernate 6.0 and 6.1 in case 6.2 metadata were used for those?

Reply:
Thank you both! Removed the defaulting for latest version. 
@sdeleuze there's metadata present that already has `"default-for": "6\\.1\\..*"`. No sure about the precedence for `6.1` if we change the value to the suggested.

Reply:
Good catch, so we should be good then.